src/call_log/call_log.c

```diff
@@ -299,4 +299,33 @@ __wt_call_log_query_timestamp(
     return (0);
 }
 
+/*
+ * __wt_call_log_begin_transaction --
+ *     Print the call log entry for the begin transaction API call.
+ */
+int
+__wt_call_log_begin_transaction(WT_SESSION_IMPL *session, const char *config, int ret_val)
+{
+    WT_CONNECTION_IMPL *conn;
+    char config_buf[128];
+
+    conn = S2C(session);
+
+    WT_RET(__call_log_print_start(session, "session", "begin_transaction"));
+    WT_RET(__wt_fprintf(session, conn->call_log_fst, "    \"session_id\": \"%p\",\n", session));
+
+    /*
+     * The begin transaction entry includes the timestamp configuration string which is copied from
+     * the original API call.
+     */
+    WT_RET(__wt_snprintf(config_buf, sizeof(config_buf), "\"config\": \"%s\"", config));
```

Comment:
This doesn't seem like a problem to me but something to consider is that the output appears as:

```
    "input" : {
        "config": "(null)"
    },
```

when a timestamp isn't passed in to begin_transaction.

Reply:
Good find @Sean04

Reply:
We could consider this scenario in the simulator application.